README.md

```diff
@@ -1 +1,158 @@
-# js-capstone
\ No newline at end of file
+
+<a name="readme-top"></a>
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] Please make sure you remove this microverse logo from your readme. It is meant to be replaced with your project logo if you have one. :pray:

Reply:
well resolved! ✔️

Reply:
This has been resolved. Thank you